skimage/feature/_texture.pyx

```diff
@@ -264,3 +264,174 @@ def _local_binary_pattern(double[:, ::1] image,
                 output[r, c] = lbp
 
     return np.asarray(output)
+
+
+cdef inline Py_ssize_t _clip(Py_ssize_t x, Py_ssize_t low,
+                             Py_ssize_t high) nogil:
+    """Clips coordinate between high and low.
+
+    Parameters
+    ----------
+    x : int
+        Coordinate to be clipped.
+    low : int
+        The lower bound.
+    high : int
+        The higher bound.
+
+    Returns
+    -------
+    x : int
+        `x` clipped between `high` and `low`.
+    """
+
+    if(x > high):
+        return high
+    if(x < low):
+        return low
+    return x
+
+
+cdef inline cnp.double_t _integ(
+    cnp.double_t[:, ::1] img, Py_ssize_t r0, Py_ssize_t c0,
```

Comment:
Fix indentation here, first parameters should be on first line after parentheses